src/main/java/dev/warrant/WarrantClient.java

```diff
@@ -267,76 +268,77 @@ public PricingTier[] listPricingTiersForUser(String userId, int limit, int page)
     }
 
     // Assign
-    public void assignRoleToUser(Role role, User user) throws WarrantException {
-        assignRoleToUser(role.getRoleId(), user.getUserId());
+    public Warrant assignRoleToUser(Role role, User user) throws WarrantException {
+        return assignRoleToUser(role.getRoleId(), user.getUserId());
     }
 
-    public void assignRoleToUser(String roleId, String userId) throws WarrantException {
-        makePostRequest("/v1/users/" + userId + "/roles/" + roleId, Collections.EMPTY_MAP);
+    public Warrant assignRoleToUser(String roleId, String userId) throws WarrantException {
+        return createWarrant(new Role(roleId, "", ""), "member", new WarrantSubject("user", userId));
```

Comment:
We should create `Role`/`Permission` constructors that let you create a `Role`/`Permission` without passing in a `name` and `description` and use those instead:
```java
// Role.java
public Role(String roleId) {
    this.roleId = roleId;
}

// Permission.java
public Permission(String permissionId) {
    this.permissionId = permissionId;
}

```

Reply:
Updated